byterun/unix.c

```diff
@@ -426,3 +426,8 @@ char *caml_secure_getenv (char const *var)
     return NULL;
 #endif
 }
+
+int caml_channel_flags(int fd)
+{
+  return 0;
+}
```

Comment:
Is this change meant to be in this PR?

Reply:
It's the Unix version of the new caml_channel_flags - previously, it was specified using `#define caml_channel_flags(fd) 0`

Reply:
Sorry if I am missing something obvious, but I don't see any other change related to `caml_channel_flags` apart from this function definition in `byterun/unix.c`.  Why is it necessary for this PR?

Reply:
Oh goodness, more :coffee: required. You are correct, sorry - it's harmless, but this function should be in the other pull request :blush: